relay/cmd/main.go

```diff
@@ -85,7 +81,11 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&cobraConfig.LetsencryptAWSRoute53, "letsencrypt-aws-route53", false, "use AWS Route 53 for Let's Encrypt DNS challenge")
 	rootCmd.PersistentFlags().StringVarP(&cobraConfig.TlsCertFile, "tls-cert-file", "c", "", "")
 	rootCmd.PersistentFlags().StringVarP(&cobraConfig.TlsKeyFile, "tls-key-file", "k", "", "")
-	rootCmd.PersistentFlags().StringVarP(&cobraConfig.AuthSecret, "auth-secret", "s", "", "log level")
+	rootCmd.PersistentFlags().StringVarP(&cobraConfig.AuthSecret, "auth-secret", "s", "", "auth secret")
+	rootCmd.PersistentFlags().StringVar(&cobraConfig.LogLevel, "log-level", "info", "log level")
+	rootCmd.PersistentFlags().StringVar(&cobraConfig.LogFile, "log-path", "console", "log path")
+
+	setFlagsFromEnvVars(rootCmd)
```

Comment:
The order usually is env -> flags, i.e. cli flags override any env vars. At least that's how aws & friends handle it. Now I must admit I'm unsure how we handle it in our other components, maybe that should take precedence.

Reply:
You are right but this code does what you expect.
In this example the listen address will be :1234
```
NB_LISTEN_ADDRESS=:654 ./cmd --listen-address=:1234 
```
But fix me if I mixed something.

Reply:
Apologies, indeed you are right. I misinterpreted how we visit the flags